packages/website/pages/blog/index.js

```diff
@@ -0,0 +1,319 @@
+import { Card, HighlightCard } from '../../components/blog/cards'
+import { useEffect, useState } from 'react'
+
+import Button from '../../components/button'
+import Loading from '../../components/loading'
+import Tags from '../../components/tags'
+import { allTags } from '../../components/blog/constants'
+import clsx from 'clsx'
+import fs from 'fs'
+import matter from 'gray-matter'
+import { usePagination } from '../../lib/usePagination'
+import { useRouter } from 'next/router'
+
+const BLOG_ITEMS_PER_PAGE = 9
+
+export async function getStaticProps() {
+  const files = fs.readdirSync('all-blogs')
+
+  let featuredImage = null
+
+  // we can create a "featured" flag if we want and feature blogs that way
+
+  /**
+   * @param {string} fn
+   */
+  const getBirthtime = (fn) =>
+    fs.statSync(`all-blogs/${fn}`).birthtime.getTime()
+
+  files.sort((a, b) => getBirthtime(b) - getBirthtime(a))
+
+  const posts = files
+    ? files
+        .filter((filename) => filename.toLowerCase() !== '.ds_store')
```

Comment:
Hopefully this is always compiled on a mac 😱 (not the case - especially in CI). It might be better to filter files that begin with `.` (hidden).

```suggestion
        .filter((filename) => !filename.startsWith('.'))
```